test/e2e/api_inheritance/api_inheritance_test.go

```diff
@@ -269,6 +276,7 @@ func TestAPIInheritance(t *testing.T) {
 
 	}, framework.KcpConfig{
 		Name: "main",
+		Args: []string{"--install-workspace-controller"},
```

Comment:
I don't believe this is necessary?

Reply:
The fix in server.go was to only set the workspace lister on the `kcpAPIExtensionsSharedInformerFactory` only if the `--install-workspace-controller` option was set.